packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -1,7 +1,112 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700');
+@import './variables';
+@import './text_utility';
+
+:root {
+  // stylelint-disable-next-line custom-property-pattern
+  --nav-container-width: 320px;
+}
+
 :local {
-  /* stylelint-disable selector-class-pattern  */
+  @keyframes fadeIn {
+    from {left: -var(--nav-container-width);}
+    to {left: 0;}
+  }
+
+  @keyframes fadeOut {
+    from {left: 0;}
+    to {left: -var(--nav-container-width);}
+  }
+
+  body {
+    font-family: 'Roboto', sans-serif;
+    margin: 0;
+  }
+
+  .layout {
+    background: linear-gradient(#eee, #999);
+    display: flex;
+    font-size: 14px;
+    height: 100vh;
+    left: 0;
+    max-width: 100vw;
+    overflow: hidden;
+    position: absolute;
+    top: 0;
+    width: 100vw;
+  }
+
   .nav {
+    height: 100%;
+    left: 0;
+    min-height: 100vh;
+    overflow-y: hidden;
+    width: var(--nav-container-width);
+    z-index: 100;
+    @media screen and (max-width: $terra-medium-breakpoint) {
+      animation: fadeOut 250ms ease-in-out;
+      display: none;
+    }
+  }
+
+  .open {
+    animation: fadeIn 250ms ease-in-out;
+    background: inherit;
     display: block;
+    height: 100%;
+    left: 0;
+    min-height: 100vh;
+    overflow: hidden;
+    position: absolute;
+    width: var(--nav-container-width);
+    z-index: 400;
+  }
+
+  .main-container {
+    height: 100vh;
+    left: var(--nav-container-width);
+    max-width: calc(100vw - #{var(--nav-container-width)});
+    overflow: scroll;
+    position: absolute;
+    text-align: left;
+    top: 0;
+    width: calc(100vw - #{var(--nav-container-width)});
+    z-index: 100;
+    @media screen and (max-width: $terra-medium-breakpoint) {
+      left: 0;
+      max-width: 100vw;
+      width: 100vw;
+    }
+  }
+
+  .logo-container {
+    margin: 30px 25px;
```

Comment:
@bjankord should this be in `rem` or `em`? What is the rule on when to use each?